noggin/controller/user.py

```diff
@@ -301,7 +301,7 @@ def user_settings_otp_delete(ipa, username):
 def user_settings_agreements(ipa, username):
     user = User(user_or_404(ipa, username))
     agreements = [
-        Agreement(a) for a in ipa.fasagreement_find(all=False) if Agreement(a).enabled
+        Agreement(a) for a in ipa.fasagreement_find(all=True) if Agreement(a).enabled
```

Comment:
with all=False here, the enabled boolean wasnt being sent back from the fasagreement_find call, so no agreements were not showing up in the agreements list, and after rebuilding the cassettes, theses tests were failing, but changing to all=true fixed the issue

Reply:
Doesn't it return all the member lists too? Maybe we should change the freeipa-fas code to return the enabled boolean as well.

Reply:
updated to this: 
```
    agreements = [
        Agreement(a) for a in ipa.fasagreement_find(all=False, ipaenabledflag=True)
    ]
```

Reply:
Oh of course, that's much better!

Reply:
fixed!